infrastructure/ess/template.yaml

```
@@ -36,9 +36,9 @@ Parameters:
     MaxLength: '41'
     AllowedPattern: '[a-zA-Z0-9]*'
     ConstraintDescription: must contain only alphanumeric characters.
-  ESSMSKName:
+  ESSMSKName2:
```

```suggestion
  ESSMSKName:
```
Can we remove the 2 from the name? Presumably this was from while you were testing things

---

It's kind of what it went up with... I could re-make a cluster with ESSMKSName

But we had a really bizzare clash where that kept resolving to `EnterpriseSolidServerMSK`
David and I had NO idea how that was happening

I know it's grim, but sticking a 2 on the end was the recommendation for expediency.
🤔 I'm not sure what the prospect is now for renaming that... If I do so it'd deploy a new cluster, right?

---

if so, i don't reckon it's worth it (takes ages and we'd need to regen k8s)